samples/bootloader/src/main.c

```diff
@@ -178,6 +201,14 @@ void main(void)
 		return;
 	}
 
+#ifdef CONFIG_SB_DISABLE_FLASH_PATCH
+	if ((NRF_UICR->DEBUGCTRL & UICR_DEBUGCTRL_CPUFPBEN_Msk) != 0x0) {
+		nrf_nvmc_write_word((uint32_t)&NRF_UICR->DEBUGCTRL,
```

Comment:
If we are not planning on using the `nrf_nvmc_write_word` function for anything else, could we not put all of this inside a function and have the function name work as documentation?

E.g. `disable_debug_control()` and then describe inside what it does. Also we can get rid of the `#ifdef` inside main()

Reply:
I'll move the `if ((NRF_UICR...` into the function and rename it, that's cleaner I agree. I'd rather keep the ifdef in main though, to show that it doesn't happen always.

Reply:
Good point, I agree.

Reply:
Done